source/renderer/app/components/widgets/forms/WalletsDropdown.js

```diff
@@ -112,9 +117,16 @@ export default class WalletsDropdown extends Component<Props> {
         };
       }
     );
-    const selectOptions = omit(props, 'options');
+    let topError;
+    let bottomError;
+    if (errorPosition === 'bottom') bottomError = error;
+    else topError = error;
+    const selectOptions = omit({ ...props, topError }, 'options');
     return (
-      <Select options={walletsData} {...selectOptions} optionHeight={62} />
+      <>
+        <Select options={walletsData} {...selectOptions} optionHeight={62} />
+        <div className={styles.error}>{bottomError}</div>
```

Comment:
The element should be presented only if `bottomError` exists. @daniloprates can you add the restriction.
`{ bottomError && <div className={styles.error}>{bottomError}</div> }`